tigerpath/majors_and_certificates/majors/POL_2018.json

```
@@ -405,10 +665,11 @@
                 },
                 {
                   "name": "Game Theory and Applications",
-                  "max_counted": 3,
+                  "max_counted": null,

```

Don't want all four to count together.
```suggestion
                  "max_counted": 3,
```